cmd/client/container.go

```diff
@@ -188,7 +204,7 @@ var listContainersCommand = cli.Command{
 		defer conn.Close()
 		client := pb.NewRuntimeServiceClient(conn)
 
-		err = ListContainers(client, context.Bool("quiet"))
+		err = ListContainers(client, context.Bool("quiet"), context.String("id"), context.String("pod"), context.String("state"))
```

Comment:
I would love a `ListOptions` struct since there are already 5 args to this func :angel:  (feel free to skip as this is cmd/client though)